### ISSUE TITLE
Sample - V3IoC - Unable to navigate to api/swagger/ui

### ISSUE THREAD

**Issue by OskarNet** [Azure/azure-functions-openapi-extension#192]
Hi

I must be missing something but I'm unable to navigate to api/swagger/ui as RenderSwaggerUI and RenderSwaggerDocument are not shown as endpoints when running the functions project locally.
I'm looking at the sample Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.V3IoC as I've got an v3 Azure Functions project.

I'm using the latest version 0.8.1-preview

**Project packages:**
  

> <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
    <LangVersion>7.2</LangVersion>
  </PropertyGroup>

>   <ItemGroup>

    <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.0.0" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.OpenApi" Version="0.8.1-preview" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.OpenApi.Core" Version="0.8.1-preview" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.ServiceBus" version="4.1.0" />
    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.3" />
    <PackageReference Include="NLog" Version="4.6.8" />
    <PackageReference Include="System.Data.SqlClient" Version="4.8.0" />
  </ItemGroup>

Any thoughts?

Regards,
Oskar

**Comment by OskarNet** [comment]
Right. I've added the extension to seven of my Azure Functions projects and most are working. Don't think it's the other packages that are the problem as I've removed them and still not working for a few of them. Continuing to investigate

**Comment by justinyoo** [comment]
Yeah, There are several issues reporting some other NuGet packages prevents the OpenAPI document from being rendered.

**Comment by KholofeloMoyaba** [comment]
@justinyoo 
Having this issue with this pair:
```
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.OpenApi" Version="1.3.0" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.ServiceBus" Version="5.6.0" />
```

Adding 
```
    <_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>
```

Also doesn't work. When I add that, the swagger renders but doesn't pick up any operations - so I get an empty swagger

For now my only option seems to use the direct ServiceBus nuget (i.e. not the function app extensions one), which isn't ideal
